src/components/views/right_panel/VerificationPanel.js

```diff
@@ -217,20 +217,20 @@ export default class VerificationPanel extends React.PureComponent {
     };
 
     _onRequestChange = async () => {
+        this.forceUpdate();
```

Comment:
I'm not sure this forceUpdate is needed at all anymore given the parent component (EncryptionPanel) watches for all phase changes and passes them in as props so would trigger a re-render anyways

Reply:
Indeed, brilliant!